<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FePEST Prior Information Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1081</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0">
     <item row="0" column="0">
      <widget class="QLabel" name="labelInputFilePath_2">
       <property name="text">
        <string>FEFLOW model: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditInputFilePath">
       <property name="text">
        <string>D:/Repositories/feflowTraining/trunk/SchulungsDVD/model/_FePest/storage/femdata/storage.fem</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="toolButtonSelectInputFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonRefresh">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1091</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_slaves">
     <attribute name="title">
      <string>Single Instance Speedup Test</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_PREDVAR">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Test sequence:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonRunSingleInstanceTest">
          <property name="text">
           <string>Perform Speed Test</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QTableWidget" name="tableWidget_singleTestResult">
          <column>
           <property name="text">
            <string>threads</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>run time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>speedup</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_sequenceSingleTest">
          <property name="text">
           <string>1,2,3,4,5,6,7,8</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="plainTextEdit_SingleTestOut"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_statistics">
     <attribute name="title">
      <string>Multi-Instance Speedup Test</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="font">
         <font>
          <family>Courier New</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>660</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_PREDVAR_2"/>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1165</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
